src/cloud-cli/cli.ts

```diff
@@ -70,6 +71,39 @@ program
     await getAppLogs(options.name, options.host, options.port);
   });
 
+const userdb = program
+  .command('userdb');
+   
+
+userdb
+  .command('create')
+  .argument('<string>', 'database name')
+  .option('-h, --host <string>', 'Specify the host', 'localhost')
+  .option('-p, --port <port>', 'Specify the port', '8080')
+  .option('-a, --admin <admin>', 'Specify the admin user', 'postgres')
+  .option('-W, --password <admin>', 'Specify the admin password', 'postgres')
```

Comment:
What does `<admin>` means here?

Reply:
admin user

Reply:
I think these are hints about the data type that the field expects (reading from the original `<string>`). The description already explains their purpose